playbooks/openstack-ses_aio_hostconfig.yml

```diff
@@ -16,24 +16,30 @@
     # Domain name to use in the deployment. Default assumes properly configured dns name.
     domainname: "{{ ansible_domain | default('openstack.local', True) }}"
   tasks:
+    - name: Ensure all the vars are loaded
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
     - name: Configure host repositories
       raw: |
-        zypper ar -f -G http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Pool/       sle12sp3-product
-        zypper ar -f -G http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Updates/    sle12sp3-update
-        zypper ar -f -G http://provo-clouddata.cloud.suse.de/repos/x86_64/SLE12-SP3-SDK-Pool/    sle12sp3-sdk-product
-        zypper ar -f -G http://provo-clouddata.cloud.suse.de/repos/x86_64/SLE12-SP3-SDK-Updates/ sle12sp3-sdk-update
-        zypper ar -f -G http://download.suse.de/ibs/SUSE:/CA/SLE_12_SP3/ SUSE-CA
-        zypper ar -f -G http://provo-clouddata.cloud.suse.de/repos/x86_64/SUSE-OpenStack-Cloud-8-Pool/ openstackcloud-8-product
+        zypper ar -f -G {{ deploy_on_openstack_repos_to_configure[item] }} {{ item }}
+      loop:
+        - SLE12SP3-product
+        - SLE12SP3-update
+        - SLE12SP3SDK-product
+        - SLE12SP3SDK-update
+        - SUSE-CA
+        - OpenStack-Cloud8-product
+        - SES5-product
+        - SES5-update
+        - SLE12Containers
+
+    - name: Refresh zypper. After this step, we can rely on ansible modules.
+      raw: |
         #TODO(evrardjp): Check if utilities/devel tools/virt/cloud8updates are really needed
         #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/utilities/SLE_12/ SLE12-utilities;
         #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/Virtualization:/containers/SLE_12_SP3/ Virtualization_containers;
         #zypper ar -f -G  http://kirk.provo.novell.com/repositories/SUSE/Products/OpenStack-Cloud/8/x86_64/product/ openstackcloud-8-update
```

Comment:
Why keep this commented lines? And, anyway, why keep repos in Refresh zypper task and not in the repo list?

Reply:
I think I would like a 100% success on the actual step 7 before removing this artifacts of old code. But I can remove it now too.

I just thought it was wise to "keep it" with the refactor. This would make git blame easy (assuming it was a failure to remove it, the gitblame could make us think it was removed by mistake in this patch, but it was removed in a previous patch already...), as it is harder to follow a chain of changes with new files :p... Anyway, wanted to keep close to the rest.